chat.js

```diff
@@ -634,16 +636,21 @@ class CommandContext {
 	 * @param {string | User} user
 	 * @param {string} note
 	 * @param {boolean} noip
+	 * @param {boolean} noalts
 	 */
-	modlog(action, user, note, noip) {
+	modlog(action, user, note, noip, noalts) {
```

Comment:
Would probably recommend an `options` object, so it's clearer what the options actually are.